the-collab-lab/tcl-71-smart-shopping-list
# Replace the loading component by one styled with tailwind

## Description

The initial loading component was styled with puur CSS. As we moved to tailwind for the styling, we add a loading component style with tailwind.


## Type of Changes

`bug fix` 

## Updates


### After
Note: To present the new loading component, I added it on the home page temporary, but there is currently no loading component on the home page.
![Screenshot 2024-04-03 124852](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/114222588/3ac31333-a053-4f1e-9377-2eeeb49a5298)

## Testing Steps / QA Criteria

- git pull origin ce-loading-tailwind
Start the server and you will be able to see the loading element when navigating to the list page.
